wiki/Contributor_Code_of_Conduct/en.md

```diff
@@ -0,0 +1,92 @@
+# Contributor Covenant Code of Conduct
+
+## Our Pledge
+
+We as members, contributors, and leaders pledge to make participation in our community a harassment-free experience for everyone, regardless of age, body
+size, visible or invisible disability, ethnicity, sex characteristics, gender identity and expression, level of experience, education, socio-economic status, nationality, personal appearance, race, religion, or sexual identity and orientation.
+
+We pledge to act and interact in ways that contribute to an open, welcoming, diverse, inclusive, and healthy community.
+
+## Our Standards
+
+Examples of behavior that contributes to a positive environment for our community include:
+
+- Demonstrating empathy and kindness toward other people
+- Being respectful of differing opinions, viewpoints, and experiences
+- Giving and gracefully accepting constructive feedback
+- Accepting responsibility and apologizing to those affected by our mistakes, and learning from the experience
+- Focusing on what is best not just for us as individuals, but for the overall community
+
+Examples of unacceptable behavior include:
+
+- The use of sexualized language or imagery, and sexual attention or advances of any kind
```

Comment:
is all of this referring to public spaces / settings? If this is for public spaces, that is something i can fully support

If it also refers to what team members do privately this can be read as "team members must not have relationships with community members" or at least extrapolated to something among those lines

And, in my opinion, so long as both parties are adults that's their own business

Reply:
I think it needs to be explained where the line is drawn in terms of any "private" settings, or any realm outside of the osu! game (discord, ect). It may be fair to assume most of the harassment that underaged people have experienced come from parties outside of the osu! game.